src/index.rs

```diff
@@ -523,6 +522,7 @@ impl Index {
     self.client.get_block(&hash).into_option()
   }
 
+  /// This gets the first inscription on a sat, with reinscriptions enabled it would return a Vec
```

Comment:
The comment is obsolete, since we now have reinscriptions. We should probably make sure here to always return the chronologically first inscription for a given sat using our ordering table.